boards/arm64/fvp_base_revc_2xaemv8a/board.cmake

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+set(EMU_PLATFORM armfvp)
+set(ARMFVP_BIN_NAME FVP_Base_RevC-2xAEMvA)
+
+set(ARMFVP_FLAGS
+  -C bp.secure_memory=0
+  -C cluster0.NUM_CORES=1
+  -C bp.refcounter.non_arch_start_at_default=1
```

Comment:
I am not sure, for FVP_Base_RevC_2xAEMv8A, whether this config is needed.

Reply:
right. I'll take that out.

Reply:
Well, after all this is actually needed. Putting it back.